CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/Modules")
 ################ PROJECT VERSION ####################
 set(PROJECT_VERSION_FULL "0.1.9")
 set(PROJECT_SO_VERSION 7)
-set(HEX_VERSION_OVERRIDE "0x108") # For CMake < 3.13
+set(HEX_VERSION_OVERRIDE "0x109") # For CMake < 3.13
```

Comment:
Ooh, thanks! This indirectly reminded me that **I** forgot to bump the `OpenShotAudio REQUIRED VERSION` in `tests/CMakeLists.txt`, when I bumped the one in `src/CMakeLists.txt` (on the develop branch).

Because `FindOpenShotAudio.cmake` is one of only like two remaining modules in my upgrade crosshairs, at which point we can finally do away with all of that duplicated library detection code once and for all.

(Don't worry, though, that's an "after-release" sort of thing. What's there is "good enough" for now. `FindRESVG.cmake` _wasn't_, as I discovered trying to use it on my local system.)

Reply:
Huh. I guess I accidentally fixed that already, as part of a different commit. Yay?